apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -78,7 +82,10 @@ const initialState: AichatState = {
   showWarningModal: true,
   chatMessageError: false,
   currentAiCustomizations: EMPTY_AI_CUSTOMIZATIONS,
+  previouslySavedAiCustomizations: EMPTY_AI_CUSTOMIZATIONS,
   fieldVisibilities: DEFAULT_VISIBILITIES,
+  viewMode: ViewMode.EDIT,
+  hasPublished: false,
```

Comment:
As mentioned in the follow-ups section of the description, this needs to be persisted along with the other model customizations. Haven't decided if we should update the `AiCustomizations` type to include it, or keep it as a separate top-level property. LMK if you have opinions!

Reply:
Don't have a strong opinion on this either - initially, I thought it should be part of AiCustomizations as it pertains to the model card. However, its value determines whether the presentation view is possible or not, so I can see that as a reason for keeping as a top-level property. Maybe slightly leaning towards top-level.

Reply:
I could go either way but my instinct is to group it with `ModelCardInfo` since it pertains to the model card and is a persisted value. Might be easiest to just add a `hasPublished` to `ModelCardInfo` and make the default `false`? That way any levelbuilder-created customizations will always start as unpublished.